static/bidder-params/taboola.json

```diff
@@ -28,6 +28,12 @@
       "items": {
         "type": "string"
       }
+    },
+    "pageType": {
+      "type": "string"
+    },
+    "position": {
+      "type": "integer"
```

Comment:
create docs PR to include description for these new bidder params in [prebid.github.io](https://github.com/prebid/prebid.github.io) repo

Reply:
@onkarvhanumante  we already did it (because at first we added these param in the prebid.js adapter )
https://docs.prebid.org/dev-docs/bidders/taboola.html

was merged from this closed PR:
https://github.com/prebid/prebid.github.io/pull/4235/files

Reply:
Thanks @ahmadlob  for clarification